libdnf/rpm/solv_query.cpp

```diff
@@ -424,22 +395,22 @@ inline static void filter_evr_internal(
 }
 
 SolvQuery & SolvQuery::ifilter_evr(libdnf::sack::QueryCmp cmp_type, const std::vector<std::string> & patterns) {
-    Pool * pool = p_impl->sack->pImpl->get_pool();
+    Pool * pool = pImpl->sack->pImpl->get_pool();
```

Comment:
May I ask you for rename or PackageSet `pImpl` to `p_impl`? Or please set up a TODO. Just to unify an internal interface.

Reply:
I added commit that unifies it in all classes (PackageSet, PackageSetIterator, ReldepList, ReldepListIterator, SolvSack)